sequencer/src/persistence/sql.rs

```diff
@@ -555,8 +555,11 @@ impl SequencerPersistence for Persistence {
     async fn record_action(
         &mut self,
         view: ViewNumber,
-        _action: HotShotAction,
+        action: HotShotAction,
     ) -> anyhow::Result<()> {
+        if !matches!(action, HotShotAction::Propose | HotShotAction::Vote) {
```

Comment:
Can we add a comment that this should be removed once we have #1931 , to prevent possible double voting across restarts?